.github/workflows/ci.yml

```
@@ -20,7 +20,7 @@ jobs:
     strategy:
       matrix:
         # Nextflow versions: check pipeline minimum and current latest
-        nxf_ver: ['20.11.0-edge']
+        nxf_ver: ['21.04.0']
```

Will be the same version twice for now, but in the future the idea is to test against the latest release and the minimum supported (`21.04.0`).
```suggestion
        nxf_ver: ['21.04.0', '']
```